lib/runner/extract-runnable-items.js

```diff
@@ -198,9 +198,9 @@ var sdk = require('postman-collection'),
         // at this point of time, we should have traversed all items mentioned in entrypoint and created a linear
         // subset of items. However, if post that, we still have items remaining in lookup object, that implies that
         // extra items were present in user input and corresponding items for those do not exist in collection. As such
-        // we need to bail out if any of the given entry-point is not found.
+        // we need to bail out with an error if any of the given entry-point is not found.
         if (Object.keys(entrypointLookup).length) {
-            return callback(null, []);
+            return callback(new Error('Some entrypoints are not found'));
```

Comment:
could you help me to throw a semantically correct error, that will be properly handled by the dependent packages?

Reply:
I've noticed there is some kind of special syntax for error message. Should we pick something similar?

<img width="895" alt="Screenshot 2021-08-07 at 12 53 27" src="https://user-images.githubusercontent.com/30000366/128596318-f708149e-3c85-410e-948b-f561f1d39e96.png">